<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAvaRMB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>可用：￥</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblRMB">
       <property name="text">
        <string>0.0000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAvaBitcoin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>可用：฿</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBitcoin">
       <property name="text">
        <string>0.0000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuy">
       <property name="text">
        <string>买</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSell">
       <property name="text">
        <string>卖</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slot_Start()</slot>
  <slot>slot_Stop()</slot>
 </slots>
</ui>
